### PR TITLE
Fix #67: Crypto 3.0: Display activation version

### LINKED ISSUE #67: Crypto 3.0: Display activation version
We should display activation version in PowerAuth admin.

### PR DESCRIPTION
**Pull request by romanstrobl**
This PR adds display of activation version in PowerAuth Admin user interface.

### CODE REVIEW

File: powerauth-admin/src/main/java/io/getlime/security/app/admin/model/SignatureAuditItem.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
     private String signature;
     private String note;
     private boolean valid;
+    private Integer version;
```
Comment (petrdvorak) [inline powerauth-admin/src/main/java/io/getlime/security/app/admin/model/SignatureAuditItem.java]: We should be very careful when storing additional data in the signature audit log. The tables are already growing rapidly. Therefore, I suggest not to put this version in it.

Instead, we should:
- Display the version by the activation, and...
- Make activation version an event in activation history table.

File: powerauth-admin/src/main/webapp/WEB-INF/jsp/activationDetail.jsp [hunk 2]
```diff
@@ -232,6 +251,12 @@
                                                         Counter<br>
                                                         <span class="black"><c:out value="${item.activationCounter}"/></span>
                                                     </td>
+                                                    <td>
+                                                        Version<br>
```
Comment (petrdvorak) [inline powerauth-admin/src/main/webapp/WEB-INF/jsp/activationDetail.jsp]: dtto

File: powerauth-admin/src/main/java/io/getlime/security/app/admin/model/SignatureAuditItem.java (hunk 1 above)
Comment (romanstrobl) [reply powerauth-admin/src/main/java/io/getlime/security/app/admin/model/SignatureAuditItem.java]: Note that we do not have an `activationStatus` for the version change (the activation history table currently only logs changes in `activationStatus`). We can introduce a new column in the table, I don't think we want to change the `activationStatus` state diagram.

**Comment by romanstrobl** [comment]
We agreed to keep the version in signature audit log and handle the database size table problem by documenting how to partition the database table.

**Pull request merged**